app/forms/work_form.rb

```diff
@@ -125,15 +125,27 @@ def create_date_range_sequence
     errors.add(:create_date_range_from, 'must be before end date')
   end
 
-  def correct_work_subtype_length
+  def validate_work_subtypes
```

Comment:
This feels sorta awkward. I'd recommend 2 separate [conditional validations](https://guides.rubyonrails.org/active_record_validations.html#conditional-validations) (one for music and one for mixed materials).

Reply:
Agree, working through it. Thanks :-)